src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -10,8 +10,8 @@ namespace System.Text.Json
     public enum JsonCommentHandling : byte
     {
         Disallow = (byte)0,
-        Allow = (byte)1,
-        Skip = (byte)2,
+        Skip = (byte)1,
```

Comment:
This is a "soft" breaking change because ideally folks are using the named enum value and not relying on casts from numeric values like int. If they are, the caller code would have to change.

Reply:
Has this gone through API review?

Reply:
I am going to get it approved offline since it is such a minor change. Any other feedback?

Reply:
Why the change?

Reply:
> Why the change?

a) It makes `JsonDocument`/`JsonSerializer` validation checks faster (since we can just do a single range check rather than adding multiple checks for the enum values that are supported)
   - This is less important now that I am not using a range check.

b) Conceptually, it is now ordered more cleanly from most commonly used to least (if you don’t want to Disallow, you likely just want to Skip).